tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "allowJs": false,
     "skipLibCheck": true,
-    "esModuleInterop": false,
+    "esModuleInterop": true,
```

Comment:
Yeah, so the `Header` using Vite runs/builds on ESM but the `js-cookie` package, now that it's added back in and is called, doesn't play nicely unless this is true.